packages/documentation/src/stories/components/dialog/dialog.stories.ts

```diff
@@ -188,7 +188,7 @@ const FormTemplate = {
                 required
               />
               <label class="form-label" for="example-dialog-text-field">Label</label>
-              <div class="form-text">
+              <div class="form-hint">
                 Hintus textus elare volare cantare hendrerit in vulputate velit esse molestie
                 consequat, vel illum dolore eu feugiat nulla facilisis.
               </div>
```

Comment:
Form hints should be connected through aria-describedby on the input field to the id on the hint and the text should be in a paragraph tag instead of a div.

```suggestion
              <p class="form-hint" id="example-form-hint">
                Hintus textus elare volare cantare hendrerit in vulputate velit esse molestie
                consequat, vel illum dolore eu feugiat nulla facilisis.
              </p>
```